tools/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ add_custom_target(all_tools)
 
 macro(add name)
   add_executable(${name} ${name}.cpp ${ARGN})
-  install(FILES ${name}.cpp DESTINATION ${CMAKE_INSTALL_DATADIR}/caf/tools)
+  install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${name} DESTINATION ${CMAKE_INSTALL_DATADIR}/caf/tools)
```

Comment:
This should either use `install(PROGRAMS $<TARGET_FILE:${name}> ...)` or `install(TARGETS ${name} ...)` so the installed target has the correct permissions. The latter is usually preferred.

Reply:
> This should either use `install(PROGRAMS $<TARGET_FILE:${name}> ...)` or `install(TARGETS ${name} ...)` so the installed target has the correct permissions. The latter is usually preferred.

Thanks for the hint. I'm not too familiar with cmake :slightly_smiling_face: